migra/formatter.py

```diff
@@ -2,16 +2,79 @@
 
 class Formatter:
 
-    def __init__(self, eager_parens: bool=True) -> None:
+    def __init__(self, eager_parens=True, destructive_func_replace=True) -> None:
         # add other format wide configs
         self.eager_parens = eager_parens
+        self.destructive_func = destructive_func_replace
```

Comment:
document these flags
for both `destructive_func` + `eager_parens` its like this fake parameter that you expose as a contract but it is needed to do _anything_ useful in each method
uneccessary complexity unless proven otherwise

Reply:
[ditto](https://github.com/latchbio/migra/pull/4#discussion_r794925783)